quicvarint/io.go

```diff
@@ -31,7 +31,10 @@ func NewReader(r io.Reader) Reader {
 
 func (r *byteReader) ReadByte() (byte, error) {
 	var b [1]byte
-	_, err := r.Reader.Read(b[:])
+	n, err := r.Reader.Read(b[:])
```

Comment:
This is obvious and good as is, but as a general note `ReadAtLeast` follows the same semantics here (not suggesting we change this here).